### ISSUE TITLE
Changing actor appearance unloaded character in game

### ISSUE THREAD

**Issue by thrdplanet** [imchillin/Anamnesis#1039]
**Describe the bug**
Opened anamnesis, switched to the actor tab, loaded a saved character appearance I have for another race (f!elezen -> f!highlander), tried to uncheck the lipstick option before the model had fully loaded, and suddenly ana said it could not connect to my character and my character did not load in the game - nothing visible in game, I was unable to move, use any actions, do anything besides rotate the camera. Closing ana did not fix the issue, I had to log out of the game. 

**Log file**
[2022-04-30-21-50-56.txt](https://github.com/imchillin/Anamnesis/files/8598315/2022-04-30-21-50-56.txt)

**Comment by Yuki-Walsh** [comment]
Are you using penumbra? I've seen similar issues with too many refreshes at once with penumbra character collections.

**Comment by thrdplanet** [comment]
No, I don't even have that installed. It also hasn't happened again with the same appearance save file, so it doesn't seem like a frequent issue or specific to that save.

**Comment by ergoxiv** [comment]
This sounds like an issue that was present prior to the memory management rework (https://github.com/imchillin/Anamnesis/pull/1394) and was caused by desynchronization between the game and Anamnesis, leading to memory corruption. I haven't heard of any reports of this happening in recent release versions so I will close this issue. Feel free to reopen it if it still occurs for you.
